projects/imagenet/experiments/custom_super.py

```diff
@@ -176,14 +179,21 @@ def my_auto_sparse_linear_params(input_size, output_size):
 # Try much longer number of epochs (with random seeds) - does do better.
 SUPER_SPARSE100_LONG = copy.deepcopy(SUPER_SPARSE100)
 SUPER_SPARSE100_LONG.update(dict(
-    epochs=80,
+    epochs=60,
+    checkpoint_freq=1,
+    keep_checkpoints_num=2,
+    checkpoint_score_attr="training_iteration",
+    checkpoint_at_end=True,
+
+    # Seed
+    seed=tune.sample_from(lambda spec: np.random.randint(2, 10000)),
```

Comment:
`tune.sample_from` without  `num_samples` is just one random seed. Was that your intention here?